lang/en/texts/data.html

```diff
@@ -150,9 +150,10 @@ <h3>If your users expect a result immediately (e.g. Nutrition apps)</h3>
 
 <h2>Discussing data, API and exports</h2>
 <p>You can <a href="https://slack.openfoodfacts.org">join the Open Food Facts Slack chatroom which is the preferred way to ask questions and discuss the API</a>.</p>
-
-<p>You can also join the mailing list to discuss the API and exports, tell us about your re-uses, and be notified when improvements or changes are made to the API. Send an empty e-mail to
+<!--
+<p>Deprecated: You can also join the mailing list to discuss the API and exports, tell us about your re-uses, and be notified when improvements or changes are made to the API. Send an empty e-mail to
 <a href="mailto:api-subscribe@openfoodfacts.org">api-subscribe@openfoodfacts.org</a> to subscribe.</p>
+-->
```

Comment:
```suggestion
```

Let's just remove this. [We have git diff](https://kentcdodds.com/blog/please-dont-commit-commented-out-code).